ui/components/Confirmation.js

```diff
@@ -0,0 +1,110 @@
+import { Button, Chip, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField, Tooltip, Typography } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import { connect } from "react-redux";
+import { closeButtonForSnackbarAction, errorHandlerGenerator, hideProgress, showProgress, successHandlerGenerator } from "./ConnectionWizard/helpers/common";
+import { pingKubernetes } from "./ConnectionWizard/helpers/kubernetesHelpers";
+import { useSnackbar } from "notistack";
+import { Search } from "@material-ui/icons";
+import { useEffect, useState } from "react";
+
+
+const styles = (theme) => ({
+  icon : {
+    display : 'inline',
+    verticalAlign : 'text-top',
+    width : theme.spacing(1.75),
+    marginLeft : theme.spacing(0.5),
+  },
+  Chip : {
+    backgroundColor : "white",
+    cursor : "pointer",
+    marginRight : theme.spacing(1),
+    marginLeft : theme.spacing(1),
+    marginBottom : theme.spacing(1),
+    height : "100%",
+  },
+})
+
+function ConfirmationMsg(props) {
+  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
+  const { classes, open, handleClose, submit, category, operation, isDelete, activeK8sContext } = props
+  const [context, setContext] = useState(activeK8sContext)
+
+  useEffect(() => {
+    setContext(activeK8sContext)
+  },[activeK8sContext])
+
+  const handleKubernetesClick = () => {
+    showProgress()
+    pingKubernetes(
+      successHandlerGenerator(enqueueSnackbar, closeButtonForSnackbarAction(closeSnackbar), "Kubernetes succesfully pinged", () => hideProgress()),
+      errorHandlerGenerator(enqueueSnackbar, closeButtonForSnackbarAction(closeSnackbar), "Kubernetes not pinged successfully", () => hideProgress())
+    )
+  }
+
+  const searchContexts = (search) => {
+    const matchedCtx = activeK8sContext.filter((ctx) => ctx.name.includes(search))
+    setContext(matchedCtx)
+  }
+  console.log(context, "state")
+  return (
+    <Dialog
```

Comment:
Could we use the MesheryPrompt component [LINK](https://github.com/meshery/meshery/blob/master/ui/components/PromptComponent.js) here also which we are using to display dialogs/modals across the UI example in Patterns/filters/applications? // @Abhishek-kumar09? This one will only remain specific to the context switcher confirmation?

Reply:
Yes, I think so